docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
 
 ### Deploying Sedna
 
-Please refer to this [link](setup/install.md).
+Sedna provides two deployment methods, which can be selected according to your actual situation:
 
-### Prepare
+- Install Sedna on a cluster Step By Step: [guide here](setup/install.md).
+- Install Sedna AllinOne : [guide here](setup/local-up.md).
 
-todo
+### Component
+Sedna consists of the following components：
 
-### Configure
+![Architecture](./proposals/images/framework.png)
 
-todo
+#### GlobalManager
+* Unified edge-cloud synergy AI task management
+* Cross edge-cloud synergy management and collaboration
+* Central Configuration Management
 
-### Run
+#### LocalController
+* Local process control of edge-cloud synergy AI tasks
+* Local general management: model, dataset, and status synchronization
 
-todo
 
-### Monitor
+#### Worker
+* Do inference or training, based on existing ML framework.
+* Launch on demand, imagine they are docker containers.
+* Different workers for different features.
+* Could run on edge or cloud.
 
-todo
+
+#### Lib
+* Expose the Edge AI features to applications, i.e. training or inference programs.
+
+
+### System Design
+
+There are three stages in a [incremental learning job](./proposals/incremental-learning.md): train/eval/deploy.
+
+![](./proposals/images/incremental-learning-state-machine.png)
+
+## Deployment Guide
+
+### 1. Prepare
+
+#### 1.1 Deployment Planning
+
+In this example, there is only one host with two nodes, which had creating a Kubernetes cluster with `kind`.
```

Comment:
`kind` is unfamiliar to non-k8s users. Besides,  do we really need to mention Kubernetes cluster and `kind` here, given that Sedna is already deployed? 
1. If not 
  Remove the sentence of "which had creating a Kubernetes cluster with `kind`" (and put a link to "Deploying Sedna" when needed).
2. If so
  2.1. Notation explaination is needed.  
  2.2. Shall it be "which is a Kubernetes cluster created with `kind`." instead of "which had creating a Kubernetes cluster with `kind`"?

Reply:
In the `prepare` chapter, `kind` is one of the ways to install sedna.